requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
+# by the following command:
 #
-#    pip-compile
+#    pip-compile requirements.in
```

Comment:
same again here? looks like some lines were not meant to be deleted?

Reply:
reverted